pkg/image/export/main.go

```diff
@@ -224,14 +224,31 @@ const (
 	linuxLoadScript = `#!/bin/bash
 images="rancher-images.tar.gz"
 list="rancher-images.txt"
+windows_image_list=""
+windows_versions="1903"
 usage () {
     echo "USAGE: $0 [--images rancher-images.tar.gz] --registry my.registry.com:5000"
     echo "  [-l|--image-list path] text file with list of images. 1 per line."
     echo "  [-i|--images path] tar.gz generated by docker save."
     echo "  [-r|--registry registry:port] target private registry:port."
+    echo "  [--windows-image-list path] text file with list of images used in Windows. Windows image mirroring is skipped when it is empty"
```

Comment:
"when it is empty" is awkward english. "when this is empty" would be better
can be changed in a separate PR